Resources/Prototypes/_Shitmed/Guidebook/medical.yml

```
@@ -0,0 +1,32 @@
+- type: guideEntry
+  id: Surgery
+  name: guide-entry-surgery
+  text: "/ServerInfo/_Shitmed/Guidebook/Medical/Surgery.xml"
+  children:
+  - PartManipulation
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)